suite-native/module-trading/src/components/general/Header/HeaderTabs.tsx

```diff
@@ -74,6 +75,22 @@ export const HeaderTabs = () => {
         selectIsFeatureFlagEnabled(state, FeatureFlag.AreTradingExchangeDexesEnabled),
     );
 
+    const onTabPress = (tab: TradingType) => {
+        if (tab === activeTab) {
+            return;
+        }
+
+        setActiveTab(tab);
+        analytics.report({
+            type: EventType.TradingNavigate,
+            payload: {
+                action: 'navigate',
+                type: tab,
+                from: 'trade',
```

Comment:
lets add more from what tab we go:

`from: 'trade' | 'trade/buy' | 'trade/sell' | 'trade/swap' | 'account' | 'dashboard';`

making it here

```suggestion
                from: `trade/${activeTab}`,
```

Reply:
Just to be sure. You want to extend the [definition](https://github.com/trezor/trezor-suite/blob/6464476043e0d9ef7f9565561c7b4511974aaba4/suite-common/analytics/src/events/suite-native/types.ts#L403) of event, right?

Reply:
Updated